docs/faq.rst

```diff
@@ -129,3 +129,54 @@ setting the ``expect`` request option to ``false``:
 
     // Disable the expect header on all client requests
     $client = new GuzzleHttp\Client(['expect' => false]);
+
+How can I track a redirected requests HTTP codes?
+============================================
+
+You can enable the `track_redirects` option which will enable Guzzle to track
```

Comment:
Maybe say "You can enable tracking the redirected URL and the status code via the ... option"